node/node.go

```diff
@@ -427,6 +429,12 @@ func NewNode(config *cfg.Config,
 
 // OnStart starts the Node. It implements cmn.Service.
 func (n *Node) OnStart() error {
+	now := tmtime.Now()
+	genTime := n.genesisDoc.GenesisTime
+	if genTime.After(now) {
+		time.Sleep(genTime.Sub(now))
```

Comment:
We should print a message like "Genesis time is in the future. Sleeping until then..."